### PR TITLE
[12.0][FIX] project_role: bad menu item

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Review by bodedra — APPROVED** [review]
LGTM :+1:

File: project_role/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     'name': 'Project Roles',
-    'version': '12.0.1.1.0',
+    'version': '12.0.1.1.1',
```
Comment (pedrobaeza) [inline project_role/__manifest__.py:6]: There's no need anymore of manual version bumping FYI

**Review by pedrobaeza — APPROVED** [review]

File: project_role/__manifest__.py (hunk 1 above)
Comment (alexey-pelykh) [reply project_role/__manifest__.py:6]: Cool! How it's decided if that's major minor or fix?

Comment (pedrobaeza) [reply project_role/__manifest__.py:6]: https://odoo-community.org/blog/the-oca-blog-1/post/the-oca-github-bot-can-now-merge-pull-requests-92

**Pull request merged**